sidebars.js

```diff
@@ -234,6 +234,7 @@ const sidebars = {
         "partners/become-a-tech-partner",
         "partners/paid-packs",
         "partners/adopt",
+        "partner/benefits",
```

Comment:
I think this is the problem
```suggestion
        "partners/benefits",
```